templates/traccar/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Output, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  services.push({
+    type: "app",
+    data: {
+      serviceName: input.appServiceName,
+      env: [].join("\n"),
+      source: {
+        type: "image",
+        image: input.appServiceImage,
+      },
+      domains: [
+        {
+          host: "$(EASYPANEL_DOMAIN)",
+          path: "/",
+          port: 8082,
+        },
+      ],
+      mounts: [
+        {
+          type: "volume",
+          name: "logs",
+          mountPath: "/opt/traccar/logs",
+        },
+        {
+          type: "file",
+          content:
+            "<!DOCTYPE properties SYSTEM 'http://java.sun.com/dtd/properties.dtd'>\r\n<properties>\r\n\r\n    <!-- Documentation: https://www.traccar.org/configuration-file/ -->\r\n\r\n    <entry key='database.driver'>org.h2.Driver</entry>\r\n    <entry key='database.url'>jdbc:h2:./data/database</entry>\r\n    <entry key='database.user'>sa</entry>\r\n    <entry key='database.password'></entry>\r\n\r\n</properties>",
```

Comment:
for multi line strings you can use  \` (backticks)

```
const traccarXML = `
<!DOCTYPE properties SYSTEM 'http://java.sun.com/dtd/properties.dtd'>
<properties>

 <!-- Documentation: https://www.traccar.org/configuration-file/ -->

 <entry key='database.driver'>org.h2.Driver</entry>
 <entry key='database.url'>jdbc:h2:./data/database</entry>
 <entry key='database.user'>sa</entry>
 <entry key='database.password'></entry>

</properties>
`;
```

Reply:
@deiucanta I have fixed and tested. Thanks!